Backend/index.js

```diff
@@ -8,14 +8,20 @@ const multer = require('multer');
 require('dotenv').config();
 
 const app = express();
+app.use(express.json());
 const prisma = new PrismaClient();
 const server = http.createServer(app);
 const wss = new WebSocket.Server({ noServer: true });
 const PORT = process.env.PORT || 5174;
 
+
 const storage = multer.memoryStorage();
 const upload = multer({ storage: storage });
 
+const PROXIMITY_RADIUS = 16093.4;
+const clients = new Map();
+
+
 server.listen(PORT, () => {
   console.log(`Server listening on http://localhost:${PORT}`);
```

Comment:
not relevant to this PR but looks like like a console.log snuck in

Reply:
This concole.log is essential because it shows me that the backend is running and that it's running on the right port.
I'll add a comment on this in subsequent PR's